diff --git a/node_modules/@react-native-community/slider/react-native-slider.podspec b/node_modules/@react-native-community/slider/react-native-slider.podspec
index b795d53..f5c4ce3 100644
--- a/node_modules/@react-native-community/slider/react-native-slider.podspec
+++ b/node_modules/@react-native-community/slider/react-native-slider.podspec
@@ -28,7 +28,7 @@ Pod::Spec.new do |s|
     s.pod_target_xcconfig = {
       "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\"",
       "OTHER_CPLUSPLUSFLAGS" => "-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1",
-      "CLANG_CXX_LANGUAGE_STANDARD" => "c++17"
+      "CLANG_CXX_LANGUAGE_STANDARD" => "c++20"
     }
     s.dependency "React-RCTFabric"
     s.dependency "React-Codegen"
